src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -803,6 +804,18 @@ public static bool ContainsDynamic(this TypeSymbol type)
         private static readonly Func<TypeSymbol, object, bool, bool> s_containsDynamicPredicate = (type, unused1, unused2) => type.TypeKind == TypeKind.Dynamic;
 
         /// <summary>
+        /// Return true if the type contains any tuples.
+        /// </summary>
+        internal static bool ContainsTuple(this TypeSymbol type) =>
```

Comment:
I think they should be the same, at least when defined in source...

---
In reply to: [69645161](https://github.com/dotnet/roslyn/pull/12344#discussion_r69645161) [](ancestors = 69645161)